### ISSUE TITLE
agario server.js problem 

### ISSUE THREAD

**Issue by Optimusprimeagario** [xzfc/agar-expose#25]
server-booted-up
{ name: 'connect_server' }
cant connect to sever

**Comment by ZfsrGhS953** [comment]
Agar.io has updated so doesn't connect to server anymore

**Comment by Optimusprimeagario** [comment]
hmmm so how can we fix it?

**Comment by ghost** [comment]
any1 fixed yet ?

**Comment by wartoz** [comment]
v-cells agarbot fixed them fb bots are back

**Comment by Optimusprimeagario** [comment]
yea but public need to fix them too

**Comment by MasonBurdette** [comment]
I fixed them, but I disagree, public should not have bots.

**Comment by Optimusprimeagario** [comment]
would you mind helping me?

**Comment by wartoz** [comment]
masonburdette i got 689 working tokens and fb bots collect pellets i have a deal for you add me on skype : iTwiGXDev

**Comment by wartoz** [comment]
http://i.imgur.com/BHTC7Wj.png

**Comment by Optimusprimeagario** [comment]
lol wartoz that not realted to this issue but i have them too

**Comment by MasonBurdette** [comment]
I'm sorry but I already have an elastic script (supports unlimited clients) and tons of cookies, so I'm all set.  I also have a farm function, taken from `agario-client`

**Comment by wartoz** [comment]
share us a way to fix this :(

**Comment by ZfsrGhS953** [comment]
we just need to load main_out.js instead of agario.core.js in index.html and add a main_out.js

**Comment by ghost** [comment]
@MasonBurdette  I also agree that public should not have bots they ruin the game and make lag with bad proxies and bad pc

**Comment by ZfsrGhS953** [comment]
@problemsofworld they should

**Comment by Nick8fr32** [comment]
any update yet??

**Comment by wartoz** [comment]
nope

**Comment by ZfsrGhS953** [comment]
A good thing is that xzfc has contributed 4 days ago, so agar-expose is probably not dead yet.

**Comment by Optimusprimeagario** [comment]
is agar bots gone completely i see no website able to fix it ?!

**Comment by Nick8fr32** [comment]
@Optimusprimeagario yes they change everything

**Comment by ZfsrGhS953** [comment]
Even Raga and Bagar are not working now, so I think bots are going to start working after them.

**Comment by Optimusprimeagario** [comment]
i successfully unpatched bots but problem is agario-client is broken ;( i request to MasonBurdette to make us updated agario-client we will appreciate your work ;)
